app/views/storage_locations/_storage_location_row.html.erb

```diff
@@ -2,6 +2,7 @@
   <td><%= storage_location.name %></td>
   <td><%= storage_location.address %></td>
   <td><%= storage_location.size %></td>
+  <td><%= storage_location.inventory_total_value %></td>
```

Comment:
This all looks great and what we're looking for to help the diaper banks. The only thing I think we might want to add is a $. So maybe `<td>$<%= storage_location.inventory_total_value %></td>`

Reply:
I'm a fan of `number_to_currency`: https://api.rubyonrails.org/classes/ActionView/Helpers/NumberHelper.html#method-i-number_to_currency :)

Reply:
Awesome. I am changing it to number_to_currency.